### DISCUSSION TITLE
Packaging as a deb file

### DISCUSSION THREAD

**Discussion by dnet890** [ciderapp/Cider#1035] [Feature Request]
Hi
Is there any possibility for deb file for this app?

**Comment by GamingLiamStudios** [comment]
It already exists? In both [self-compilation on Linux systems](https://docs.cider.sh/compilation/compiling-on-linux) & in the [official binary releases](https://github.com/ciderapp/cider-releases/releases), there is a `.deb` package.

**Comment by dnet890** [comment]
ah yes. But, there is no ETA for stable release. Is that so?

**Reply by cryptofyre** [reply]
There is a lot of work that need’s to be done before Cider can be considered “stable”. Due to current circumstances we are kind of working in limited capacity.
